server/pulp/server/webservices/views/serializers/__init__.py

```diff
@@ -0,0 +1,163 @@
+import copy
+
+from django.core.urlresolvers import reverse
+
+
+class BaseSerializer(object):
+    """
+    Base class to be used for creating serializers
+    """
+
+    def __init__(self, instance=None, context=None, multiple=False):
```

Comment:
I *instance* really an optional parameter?

Would it be reasonable to detect if *instance* is iterable instead of relying to *multiple* being passed correctly?